dcc-portal-server/src/main/java/org/icgc/dcc/portal/server/model/File.java

```diff
@@ -77,9 +89,6 @@ public static File parse(@NonNull String json) {
 
   }
 
-  @ApiModelProperty(value = "Data bundle info of a repository file")
```

Comment:
They are, I just reorganized the field declarations to be a bit more condensed. See line 69.